### PR TITLE
Initial draft for ingest management overview

### PR DESCRIPTION
**Pull request by mostlyjason**
Adding initial content for the overview. It's not in full asciidoc format yet. Also, we may need to rearrange or refactor some content.

### CODE REVIEW

File: docs/en/ingest-management/ingest-management-overview.asciidoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+=== Configuring Integrations
+
+We provide a web-based UI for configuring integrations with your data sources. This can include popular services and platforms like Nginx or AWS, as well as many generic input types like log files.
+
+The Elastic Agent configuration allows you to configure any number of integrations for data sources. The Agent configuration can be applied to multiple Agents. This makes it even easier to manage configuration at scale.
+
+[role="screenshot"]
+image::images/data-source.png[Configuring a data source]
+
+You may define a data source by supplying a name and description. You can then configure inputs for logs and metrics, such as the path to your Nginx access logs. When you are done, you may save the data source to update the Agent configuration. The next time enrolled Agents check in, they will receive the update. Having those configurations automatically deployed is more convenient than doing it yourself using SSH, Ansible playbooks, etc.
```
[the remark this answers is not in the document]
Comment (mostlyjason) [reply docs/en/ingest-management/ingest-management-overview.asciidoc:30]: Feel free to change it. I usually think of an overview being descriptive whereas a how-to guide would be a series of steps.

[the remark this answers is not in the document]
Comment (dedemorton) [reply docs/en/ingest-management/ingest-management-overview.asciidoc:30]: Agreed. We don't want to cover steps in this overview. We want to introduce concepts and explain how the parts work together.  I'll take a pass at this when I do the edit. I do understand why you included this here.

**Comment by dedemorton** [comment]
BTW, don't worry about cleaning up the asciidoc formatting. I can do that. Nothing will break until I hook the content up to the build.

**Comment by dedemorton** [comment]
I've just hooked up the docs to the build, but I'm [commenting out](https://github.com/elastic/stack-docs/pull/1157) the overview include temporarily until you've merged this PR and I can work on the asciidoc tagging. I made some structural changes that will impact your PR.

**Pull request merged**